client/app/components/book-item-in-collection/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Why not just book or book item. So you can use it outside of the collection without the name making no sense